src/runtime_src/core/tools/xbutil2/SubCmdValidate.cpp

```diff
@@ -432,14 +432,77 @@ p2ptest_chunk(xclDeviceHandle handle, char *boptr, uint64_t dev_addr, uint64_t s
   return true;
 }
 
+static bool
+p2ptest_chunk_no_dma(xclDeviceHandle handle, xclBufferHandle bop2p, size_t bo_size, int bank)
+{
```

Comment:
a comment here would be helpful. Something like:
// testing p2p write flow host -> device